### DISCUSSION TITLE
Apollo Playground not working for router

### DISCUSSION THREAD

**Discussion by mohit61** [apollographql/router#1960] [General]
Hi, 

I am running router in local at port 7777 and at path "/graphql", but the playground is not working. I get the below screen on opening the `http://localhost:7777/graphql` in the browser

<img width="1673" alt="Screenshot 2022-10-17 at 4 34 26 PM" src="https://user-images.githubusercontent.com/32329331/196161953-508f2c41-e3d9-44c5-bf71-f4648c2470cd.png">

**Comment by bnjjj** [comment]
I guess what you call the playground is the sandbox. By default it's disabled, to enable it take a look at our documentation here https://www.apollographql.com/docs/router/configuration/overview let me know if it helps

**Comment by mohit61** [comment]
Thanks.
